test/general/fork.pass.cpp

```diff
@@ -0,0 +1,13 @@
+#include <oneapi/dpl/execution>
+#include "support/utils.h"
+
+#if __has_include(<unistd.h>)
+#include <unistd.h>
+#endif
+
+int main(){
+#ifdef _POSIX_VERSION // defined in <unistd.h>
+    [[maybe_unused]] pid_t pid = fork();
```

Comment:
Would it be good to wait in the parent process and check the child process' exit status to make sure nothing went wrong?

Reply:
Why not; added.